# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/ChromeUtils.cpp
# Commit: 5598943a94fd
# Full Hash: 5598943a94fd9e6be45bbc7125ef7bac49a4bf05
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2021-12-15 21:51:13
# Regressor Bug: 1744025
# File Overlap Count: 3
# Description:
#   Bug 1744025: Use CheckedUnsafePtr<WorkerPrivate> in WorkerGlobalScopeBase. r=dom-worker-reviewers,asuth
#   
#   Depends on D132708
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D132709
# ==============================================================================

diff -r 43186fbbf8b4 -r 5598943a94fd dom/base/ChromeUtils.cpp
--- a/dom/base/ChromeUtils.cpp	Wed Dec 15 13:48:17 2021 +0000
+++ b/dom/base/ChromeUtils.cpp	Wed Dec 15 13:48:17 2021 +0000
@@ -42,7 +42,7 @@
 #include "mozilla/dom/UnionTypes.h"
 #include "mozilla/dom/WindowBinding.h"  // For IdleRequestCallback/Options
 #include "mozilla/dom/WindowGlobalParent.h"
-#include "mozilla/dom/WorkerPrivate.h"
+#include "mozilla/dom/WorkerScope.h"
 #include "mozilla/ipc/GeckoChildProcessHost.h"
 #include "mozilla/net/UrlClassifierFeatureFactory.h"
 #include "IOActivityMonitor.h"